app/grandchallenge/algorithms/serializers.py

```diff
@@ -110,13 +110,14 @@ class Meta:
         model = Job
         fields = [
             "pk",
+            "url",
             "api_url",
             "algorithm_image",
+            "algorithm",
             "inputs",
             "outputs",
             "status",
             "rendered_result_text",
-            "algorithm_title",
```

Comment:
I'm not sure that algorithm_title can be removed. Won't it need to be deprecated and go through two CIRRUS release cycles first?

Reply:
Hmm, it was added in 2020 without any context, any ideas why @HarmvZ ? https://github.com/comic/grand-challenge.org/pull/1633

Reply:
https://github.com/DIAGNijmegen/rse-cirrus-core/commit/c33fa4b6a8fd0f44b184b54c5ed709dbe5dac84f

That plugin has seen major changes since.

Reply:
Great, and that was removed in https://github.com/DIAGNijmegen/rse-cirrus-core/commit/de699e337cb10a7588afc3cd80f97375f911e193#diff-b3e0218cb1d9d5340a73e38d6226ea7dc783dd8784f94ed749f4c766c957b0b1.

Reply:
FYI. This did show up in the predictions.json, but I do not think anyone is actively reliant upon it. Or I would hope not...